docs/test-suites.md

```diff
@@ -280,4 +281,53 @@ Name | Type
 `useRenderer` | Boolean
 `useRobotsOverwrite` | Boolean
 
-<!-- tabs:end -->
\ No newline at end of file
+<!-- tabs:end -->
+
+## Global Templates
+
+This feature enables you to link two or multiple test suites together, by making one of it a global template. When any of the global template configuration in Step 2 or any test configuration in Step 3 is updated, all the linked test suites configurations and test configurations will be updated.
```

Comment:
Please mention what is not copied. We have some exceptions.

Reply:
hmm, to be honest, I don't remember the exceptions, that's why I didn't mention them

Reply:
crawl type is always set to web.  ( so not file upload also)
startURL is also not copied.